scripts/__tests__/build-blog-data-sanitization.test.js

```
@@ -0,0 +1,475 @@
+
+    test('should handle deeply nested content', () => {
+      const deeplyNested =
+        '<div>'.repeat(100) + 'Content' + '</div>'.repeat(100)
+      const sanitized = sanitizeHtml(deeplyNested, sanitizeOptions)
+
+      expect(sanitized).toContain('Content')
+      // Should not cause stack overflow or other issues
+    })
+
+    test('should handle very long content', () => {
+      const longContent = '<p>' + 'A'.repeat(10000) + '</p>'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)